include/Track.h

```diff
@@ -127,7 +127,7 @@ class LMMS_EXPORT Track : public Model, public JournallingObject
 	void deleteClips();
 
 	int numOfClips();
-	Clip * getClip( int clipNum );
+	auto getClip(std::size_t clipNum) -> Clip*;
```

Comment:
I don't really understand why we use trailing type with auto instead of the normal function syntax.

Reply:
This was discussed a while ago [on Discord](https://discord.com/channels/203559236729438208/207159523616489473/1062400367184314398), and we agreed we wanted to use this style going forward.